### DISCUSSION TITLE
WebFollow with libre3 not working

### DISCUSSION THREAD

**Discussion by ralf-ms** [NightscoutFoundation/xDrip#2236] [Q&A] [answered]
I'm on my 2nd Libre 3 sensor now and used the webfollower service successfully with the first one (libre 1 sensor in between, so the 1st L3 sensor was about 2 weeks ago).
When I now use "WebFollower" in xdrip, I see the following error messages in the log (and no bg data): 
Got failure response code: 401 {"message": "MissingCachedUser"}
Any ideas?
Login-user and password for the follower-app are correct, I switched to LLU Client which is working but don't want to have another process sucking at my phones battery...

**Comment by jamorham** [comment, marked as answer]
Does switching the `Use proxy server` preference option on and then off again resolve the problem? (Any change to the settings will cause some internal resetting)

**Reply by ralf-ms** [reply]
Yes, that solved the problem!
Thanks a lot!

**Reply by jamorham** [reply]
If possible could you also use the event viewer log upload facility so that perhaps this process can be automated in future.

**Reply by ralf-ms** [reply]
done
